index.bs

```diff
@@ -1280,50 +1299,28 @@ dictionary Dimensions {
 
 See <a href="#diagram-player-startcreative">Diagram 9. Normal `Player:startCreative` Sequence</a>.
 
-The player posts `SIMID:player:startCreative` when it is ready to make the iframe visible. Under a typical flow, the player waits for a [[#simid-player-startCreative-resolve]] to display the SIMID iframe and to start media playback.
+The player posts `SIMID:Player:startCreative` message when it is ready to make the iframe visible. The player must transmit `Player:startCreative` as close to the first media frame rendering as possible. The player waits for a [[#simid-player-startCreative-resolve]] response to display the SIMID iframe. The interactive creative should be ready to reply to `Player:startCreative` immediately.
 
-In the circumstances that require uninterrupted media playback, the player posts `Player:startCreative` in the middle of the ad experience. See [[#simid-player-init]], section "Special Cases". 
+[[#simid-player-init]] section describes the flow that precedes the instant the player emits a `Player:startCreative` message.  
 
 <div diagram id="diagram-player-startcreative">
 	<p>Normal `Player:startCreative` Sequence</p>
 	<img src="images/dgrm-startCreative-normal.png" alt="startCreative normal sequence.">
 	<ol dgrm-details>
+		<li>Media playback begins.</li>
 		<li>Player posts `Player:startCreative`.</li>
 		<li>Creative responds with a `resolve`.</li>
 		<li>Player displays the creative iframe.</li>
-		<li>Player starts ad media playback.</li>
 	</ol>
 </div>
 
-
 #### resolve #### {#simid-player-startCreative-resolve}
-By posting `resolve`, the creative acknowledges that it is ready for display. The player makes the iframe visible immediately upon a `resolve` receipt (<a href="#diagram-player-startcreative">diagram 9</a>,  <span steps><b>2</b></span>).
+By posting `resolve`, the interactive creative acknowledges that it is ready for display. The creative should be ready to respond immediately. The player makes the iframe visible upon a resolve receipt (<a href="#diagram-player-startcreative">diagram 9</a>, <span step><b>2</b></span>).
```

Comment:
Can we leave out the line about the creative being ready to respond immediately, for now.

I can imagine a case where the creative just hasn't finished loading all its assets and would like to hold its response.